scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -0,0 +1,70 @@
+// CNF-1500 assembly
+[id="ztp-deploying-disconnected"]
+= Deploying distributed units at scale in a disconnected environment
+include::modules/common-attributes.adoc[]
+:context: ztp-deploying-disconnected
+
+toc::[]
+
+Telco edge computing presents extraordinary challenges with managing hundreds to tens of thousands of clusters in hundreds of thousands of locations. These challenges require fully-automated management solutions with, as closely as possible, zero human interaction.
```

Comment:
This context is great, but I think it needs to be in a concept module. The intro text in an assembly should be short, focused, and describe what the user's going to accomplish in the assembly. (It should be like the two-sentence description that you see on a link in a search results page.)